NDTensors/ext/NDTensorsMetalExt/adapt.jl

```diff
@@ -6,6 +6,6 @@ end
 
 # More general than the version in Metal.jl
 function Adapt.adapt_storage(arraytype::Type{<:MtlArray}, xs::AbstractArray)
-  arraytype_specified = specify_parameters(arraytype, get_parameters(xs))
+  arraytype_specified = specify_defaults(arraytype)
```

Comment:
This seems to change the meaning of this function, shouldn't it still be setting parameters based on those of `xs`?

Reply:
Yes, I updated the code fix this. I had originally written `specify_parameters` incorrectly and have since fixed it (I was missing exactly the functionality you described here) Thanks!